sonobi-htb.js

```diff
@@ -126,7 +126,8 @@ function SonobiHtb(configs) {
         /* build data */
         var data = {
             key_maker: JSON.stringify(keyMaker), // jshint ignore:line
-            cv: 'sbi'
+            cv: 'sbi',
+            vp: Browser.getDeviceTypeByUserAgent()
```

Comment:
Browser class doesn't have this method, please use `SpaceCamp.DeviceTypeChecker.getDeviceType()` instead. 
The method will return either `mobile` or `desktop` (tablets are considered to be desktop) by checking the user agent unless specified by a publisher to be hard coded into the wrapper.